exonum/src/runtime/mod.rs

```diff
@@ -194,6 +194,25 @@ impl From<RuntimeIdentifier> for u32 {
     }
 }
 
+impl RuntimeIdentifier {
```

Comment:
What do you think about this crate?
https://crates.io/crates/num-derive
It could be helpful to avoid of handwritten implementations.

Reply:
Looks good, but I'm not sure if we want to add new dependencies for this case.

Reply:
Agree, adding a new dependency just to save several lines is an overkill.

Reply:
The similar code is not only present here in this crate. But this is just a little remark.

Reply:
You can create an issue and mark it for the next grooming, so we'll discuss whether it worth it :)